code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -281,7 +281,7 @@
 	var/busy_icon_state
 	var/locked = FALSE
 
-	var/datum/weakref/bci_to_implant
+	var/obj/item/organ/internal/cyberimp/bci/bci_to_implant
```

Comment:
If you are going to change this from a weakref you must also implement `/Exited` on bci_implanter so that when the bci_to_implant is removed, this variable becomes null, as to not cause hard deletes.

Reply:
Got it.